src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1487,8 +1483,10 @@ def enable_features(cmd, client, resource_group_name, cluster_name, features, ku
         cmd_helm_upgrade.extend(["--kube-context", kube_context])
     if enable_azure_rbac:
         cmd_helm_upgrade.extend(["--set", "systemDefaultValues.guard.enabled=true"])
-        cmd_helm_upgrade.extend(["--set", "systemDefaultValues.guard.clientId={}".format(azrbac_client_id)])
-        cmd_helm_upgrade.extend(["--set", "systemDefaultValues.guard.clientSecret={}".format(azrbac_client_secret)])
+        # Setting the default authnMode mode as "arc" for guard. This mode used Arc RBAC 1P apps for authN/authZ.
+        # This mode in guard uses PoP token based auth.
+        cmd_helm_upgrade.extend(["--set", "systemDefaultValues.guard.authnMode=arc"])
+        logger.warning("Please use the latest kubelogin version which has support for generating PoP token(s) needed by guard running in 'arc' authN mode.")
```

Comment:
Do we need to add a specific minimum version here? Not sure but can discuss

Reply:
+1, let's be specific and asking for version "0.0.32" ? and up @rharpavat please confirm the version here

Reply:
Confirming that minimum version required for kubelogin is `v0.0.32`.